hack/generate-everything.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+run_gen() {
+    echo -e "\n=== $* ===\n"
+    "$@"
+}
```

Comment:
It might be easier to set `-o xtrace`, perhaps in combination with setting `$PS4` if a custom marker is needed.

Reply:
Yup, changed to just use `xtrace`